.github/workflows/build-python.yml

```diff
@@ -60,11 +60,11 @@ jobs:
         langchain_minimal: [false]
         # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
         include:
-          - python-version: "3.10"
-            os: windows-2019
-            pydantic_v1: false
-            pandas_v1: false
-            langchain_minimal: false
+          # - python-version: "3.10" # Deactivating windows-2019, since it's trying to use python 3.7 to install PDM. Maybe try to reactivate later ?
```

Comment:
let's delete it as long as there's windows-2022 that remains and nobody asked for 2019 yet